components/inputs/test/input-date.visual-diff.html

```diff
@@ -15,26 +15,34 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 		<meta http-equiv="X-UA-Compatible" content="ie=edge">
 		<meta charset="UTF-8">
+		<style>
```

Comment:
I did a re-order of these to be alphabetical and have simpler names.
I increased the padding to avoid the other `input-text`s showing up in the visual-diff screenshots and causing a ton of false failures. I can remove this in a subsequent PR where we aren't necessarily concerned with these false failures.